backtester/config/examples/dca-api-candles-multiple-currencies.strat

```diff
@@ -32,7 +32,8 @@
    "max-slippage-percent": 0,
    "maker-fee-override": 0.001,
    "taker-fee-override": 0.002,
-   "maximum-holdings-ratio": 0
+   "maximum-holdings-ratio": 0,
+   "use-exchange-order-limits": false
```

Comment:
Tested and working well! 

![test](https://user-images.githubusercontent.com/4685270/112395657-f0b28c80-8d52-11eb-965a-169df553654f.jpg)

Two changes before merging this in would be:

- Outputting this setting to the backtester console and HTML report, as you've already done in the config builder.
- A warning for exchanges which support order limits, but have it disabled should warn the user that the results are prone to errors (order execution won't work when in production).

Reply:
Could the message become "may not work when in production"? A user can already set their own min/max buying rules and there may have been no issue at all even if they were turned on

Reply:
Yes!